assembly-plugin-boilerplate/pom.xml

```diff
@@ -0,0 +1,52 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+ xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+ http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.confluent</groupId>
+        <artifactId>common</artifactId>
+        <version>4.0.1-SNAPSHOT</version>
```

Comment:
It looks like you're merging common-docker's 4.0.x branch into common's master branch (4.0.1-SNAPSHOT vs. 4.1.0-SNAPSHOT) -- it would be good to double check if there are any changes in common-docker master which are not in 4.0.x, and could be getting dropped by this move.

Reply:
Ah sure, 
common-docker defaulting to 4.0.x on git.
common is defaulting to master.

I will double check.